server/src/main/java/org/opensearch/index/IndexSettings.java

```diff
@@ -85,7 +85,10 @@
 public final class IndexSettings {
     private static final String MERGE_ON_FLUSH_DEFAULT_POLICY = "default";
     private static final String MERGE_ON_FLUSH_MERGE_POLICY = "merge-on-flush";
-
+    public static final String TIERED_MERGE_POLICY = "tiered";
+    public static final String LOG_BYTE_SIZE_MERGE_POLICY = "log_byte_size";
+    private static final String DEFAULT_MERGE_POLICY = "default_merge_policy";
```

Comment:
Why do we need explicit `DEFAULT_MERGE_POLICY`? We could use `TIERED_MERGE_POLICY` a s the default value to preserve current merge policy.

Reply:
I'm using these default constants to differentiate if the property is explicitly set or it's the default value which needs to be used.

Reply:
> I'm using these default constants to differentiate if the property is explicitly set or it's the default value which needs to be used.

Agree, the `default` (as the constant) should be more than enough I think